src/Akka.Streams.Kafka/Stages/Consumers/Abstract/BaseSingleSourceLogic.cs

```diff
@@ -26,10 +26,30 @@ internal abstract class BaseSingleSourceLogic<K, V, TMessage> : GraphStageLogic
         private readonly IMessageBuilder<K, V, TMessage> _messageBuilder;
         private int _requestId = 0;
         private bool _requested = false;
+        protected ISubscription Subscription { get; }
```

Comment:
We store the `ISubscription` as a protected property now because it gets used in the `ConfigureSubscription` method.